about.html

```diff
@@ -55,10 +55,10 @@ <h3 class="title">
         <div class="col col-lg-6">
           <div class="description">
             <h2>
-              Contributions are valued the same way
+              Community contributions

```

Comment:
This is a change in the value.

Reply:
I don't understand what that means? H2s do not need to be complete statements, they tell the reader what follows...which would be examples of valued community contributions yes?